docs/src/main/sphinx/admin/properties-query-management.rst

```diff
@@ -49,6 +49,18 @@ joins, aggregations, partitioned window functions and others.
 The minimum number of partitions to use for processing distributed operations, such as
 joins, aggregations, partitioned window functions and others.
 
+``query.max-writer-tasks-count``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-integer`
+* **Default value:** ``100``
+* **Session property:** ``max_writer_tasks_count``
+
+The planner will make its best effort to limit maximum number of tasks that will take part
+in writing data for executing queries such as ``INSERT``, ``CREATE TABLE AS SELECT`` and others.
+For writing to unpartitioned tables it requires ``redistribute-writes`` or ``scale-writers``
```

Comment:
`it` -> `limiting of writing tasks requires`.

Add a comment that it doesn't work with task retries